pwiz_tools/Shared/Common/SystemUtil/PathEx.cs

```diff
@@ -189,7 +189,7 @@ public static string FindExistingRelativeFile(string relativeFilePath, string fi
         /// </summary>
         public static string RemovePrefix(string path, string prefix)
         {
-            if (path.StartsWith(prefix))

```

Comment:
We don't actually know whether the file system is case sensitive or not.
It would be better to normalize both the paths before they get passed into this function.
Could you call something like "Path.GetFullPath" on the two paths before you call this function?

Reply:
When I wrote this, AFAIK Windows and Wine are always case insensitive. But reading [this](https://learn.microsoft.com/en-us/windows/wsl/case-sensitivity) it seems Windows added some ability to specify directories as case sensitive in Windows 10 build 17107. That's kind of neat but I don't think it's ever been turned on by default.

GetFullPath is case preserving so I don't see how that would make a difference:
```
Path.GetFullPath("c:/foo/Bar")
"c:\\foo\\Bar"
Path.GetFullPath("c:/foo/BAR")
"c:\\foo\\BAR"
```
Not slash preserving though it seems. :) (I tried with backslashes in the input and it was same output)

Reply:
I think the most likely way this could affect the user experience is when the user is passing in absolute paths with varying capitalization (likely in the drive letter; directory names are probably entered pretty consistently). If we expect only the drive letter to vary, then we could limit the case insensitive comparison to the first letter if the path is absolute. That should mostly avoid any case-sensitive/insensitive filesystem issues.